commands/update_organization_setcredentials.go

```diff
@@ -0,0 +1,351 @@
+package commands
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/fatih/color"
+	"github.com/giantswarm/gsclientgen/models"
+	"github.com/giantswarm/gsctl/client/clienterror"
+	"github.com/giantswarm/gsctl/config"
+	"github.com/giantswarm/microerror"
+	"github.com/spf13/cobra"
+)
+
+const (
+	updateOrgSetCredentialsActivityName = "set-org-credentials"
+)
+
+var (
+
+	// UpdateOrgSetCredentialsCommand performs the "update organizatio set-credentials" function
+	UpdateOrgSetCredentialsCommand = &cobra.Command{
+		Use:     "set-credentials",
+		Aliases: []string{"sc"},
+		Short:   "Set credentials of an organization",
+		Long: `Set the credentials used to create and operate the clusters of an organization.
+
+Setting credentials of an organization will result in all future cluster
+being run in the account/subscription referenced by the credentials. Once
+credentials are set for an organization, this cannot be undone.
```

Comment:
`this currently cannot be undone` sounds better, we probably will support this eventually